README.md

```diff
@@ -263,6 +265,37 @@ provided.  Libraries for JavaScript, TypeScript, and Elm generated by the
 [QuickType](https://github.com/quicktype/quicktype) tool are provided.  See the
 HOWTO for instructions on updating these schemas.
 
+## Kaitai Struct Format Descriptions
+
+[Kaitai Struct](https://kaitai.io/) is a declarative language used to describe
+binary data structures.  KSY files containing format specifications for all
+supported SBP messages can be generated using the `kaitai` target.  The
+resulting files can be found in the [`kaitai/ksy`](./kaitai/ksy) directory.
+This target also runs the Kaitai Struct compiler to generate parsing code for
+the following targets, which can be found in their respective subdirectories:
+
+* graphviz
+* csharp
+* perl
+* java
+* go
+* cpp_stl
+* php
+* lua
+* python
```

Comment:
Would like to reduce this to just emitting Python, if other languages are needed, presumably one can use the .ksy files to produce a parser?

Reply:
We've had customers asking about C# bindings in the past so I'd like to leave those in.  And ruby/go/lua are fairly common, and we have nothing else to offer users of these languages.  But I wouldn't have any issues with removing the following:

* graphviz
* perl
* java
* cpp_stl
* php
* nim
* html
* javascript

Reply:
Anything where we already have a language implementation should definitely be omitted unless (like Python) there is a clear advantage to the Kaitai alternative.  For other languages I am mostly worried about support, anything that we "officially" generate should either have a test suite or it should be painfully obvious that it's not officially unsupported (having to generate the language bindings yourself should make it obvious that we don't support it).  In general though, we must have a test suite for the devinfra team to be able to support the code.

Reply:
So perhaps "C# support" should come as set of documentation for how to generate the bindings and a disclaimer that it's not officially supported (yet).

Reply:
I'm fine with writing some test cases for C#, although then it would be a question of how we can automatically execute them in CI (I'm guessing mono?).

Reply:
> I'm fine with writing some test cases for C#, although then it would be a question of how we can automatically execute them in CI (I'm guessing mono?).

Note the current test harnesses generate test cases for all `test_*.yaml` files for each language, similar to how we generate language support bindings.  For example see:
- https://github.com/swift-nav/libsbp/blob/master/spec/tests/yaml/swiftnav/sbp/signing/test_MsgEcdsaSignature.yaml
- https://github.com/swift-nav/libsbp/blob/master/generator/sbpg/targets/resources/rust/test/sbp_tests_template.rs

We have access to Windows nodes, but Mono should work too, it's already available on GHA Linux images:

![image](https://github.com/swift-nav/libsbp/assets/183436/efa54464-d0ed-4487-9b22-60c0c49d07a3)

See https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md